test/routes/api/resources.js

```diff
@@ -21,7 +21,15 @@ describe('/api/resources', () => {
       const res = response.body;
       assert.deepStrictEqual(res.length, 2);
     });
+
+    it('returns a list of Resources that belong to a given Category', async () => {
```

Comment:
Here we want to test getting only the Resources that belong to a specific Category.